Description: Upstream changes introduced in version 0.10.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 haskell-shakespeare-css (0.10.0-1) unstable; urgency=low
 .
   * Initial release.
 .
 The person named in the Author field signed this changelog entry.
Author: Clint Adams <clint@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ haskell-shakespeare-css-0.10.0/1
@@ -0,0 +1,81 @@
+Source: haskell-shakespeare-js
+Section: haskell
+Priority: extra
+Maintainer: Clint Adams <clint@debian.org>
+Build-Depends: debhelper (>= 7)
+  , cdbs
+  , haskell-devscripts (>= 0.8)
+  , ghc
+  , ghc-prof
+  , libghc-parsec3-dev
+  , libghc-parsec3-prof
+  , libghc-shakespeare-dev (>> 0.10)
+  , libghc-shakespeare-dev (<< 0.11)
+  , libghc-shakespeare-prof
+  , libghc-text-dev (>> 0.7)
+  , libghc-text-dev (<< 0.12)
+  , libghc-text-prof
+Build-Depends-Indep: ghc-doc
+  , libghc-parsec3-doc
+  , libghc-shakespeare-doc
+  , libghc-text-doc
+Standards-Version: 3.9.2
+Homepage: http://hackage.haskell.org/package/shakespeare-js
+Vcs-Git: git://git.debian.org/collab-maint/haskell-shakespeare-js.git
+Vcs-Browser: http://anonscm.debian.org/gitweb/?p=collab-maint/haskell-shakespeare-js.git;a=summary
+
+Package: libghc-shakespeare-js-dev
+Architecture: any
+Depends: ${haskell:Depends}
+  , ${shlibs:Depends}
+  , ${misc:Depends}
+Recommends: ${haskell:Recommends}
+Suggests: ${haskell:Suggests}
+Provides: ${haskell:Provides}
+Description: compile-time JavaScript templates
+ This package provides a library for the Haskell programming language.
+ See http://www.haskell.org/ for more information on Haskell.
+ .
+ Shakespeare is a template family for type-safe, efficient templates
+ with simple variable interpolation . Shakespeare templates can be
+ used inline with a quasi-quoter or in an external file. Shakespeare
+ interpolates variables according to the type being inserted.
+ .
+ In this case, the variable type needs a ToJavascript instance.
+
+Package: libghc-shakespeare-js-prof
+Architecture: any
+Depends: ${haskell:Depends}
+  , ${misc:Depends}
+Recommends: ${haskell:Recommends}
+Suggests: ${haskell:Suggests}
+Provides: ${haskell:Provides}
+Description: compile-time JavaScript templates; profiling libraries
+ This package provides a library for the Haskell programming language,
+ compiled for profiling.
+ See http://www.haskell.org/ for more information on Haskell.
+ .
+ Shakespeare is a template family for type-safe, efficient templates
+ with simple variable interpolation . Shakespeare templates can be
+ used inline with a quasi-quoter or in an external file. Shakespeare
+ interpolates variables according to the type being inserted.
+ .
+ In this case, the variable type needs a ToJavascript instance.
+
+Package: libghc-shakespeare-js-doc
+Section: doc
+Architecture: all
+Depends: ${misc:Depends}, ${haskell:Depends}
+Recommends: ${haskell:Recommends}
+Suggests: ${haskell:Suggests}
+Description: compile-time JavaScript templates; documentation
+ This package provides the documentation for a library for the Haskell
+ programming language.
+ See http://www.haskell.org/ for more information on Haskell.
+ .
+ Shakespeare is a template family for type-safe, efficient templates
+ with simple variable interpolation . Shakespeare templates can be
+ used inline with a quasi-quoter or in an external file. Shakespeare
+ interpolates variables according to the type being inserted.
+ .
+ In this case, the variable type needs a ToJavascript instance.
